swagger/relay/swagger.yaml

```diff
@@ -1,36 +1,36 @@
 swagger: '2.0'
 info:
-  title: Mailgun Relayery Forward Server API
+  title: Mailgun Relayery Relay Server API
```

Comment:
I'd strip "Mailgun Relayery" since it is evident in the repository/application name? (The same applies to the control API.)